types/app.d.ts

```diff
@@ -168,6 +176,24 @@ declare namespace Realm {
             password: string,
         ): Credentials<Credentials.EmailPasswordPayload>;
 
+        /**
+         * Creates credentials that logs in using the [Custom Function Provider](https://docs.mongodb.com/realm/authentication/custom-function/).
```

Comment:
Disclaimer: I am not a native English speaker.

`that logs` -> `to log`

Reply:
I changed it to "authenticates".

Reply:
I think those are all correct, but expressing slightly different aspects of the idea. I would probably phrase it as "for logging in", but none of these are really wrong.

More importantly, though, I think we need to change the `Creates credentials` part. With a lower-case, non-code-font "credentials", it implies that you are making new credentials, ie making a new account, or generating a new API key, but I don't think that is what this does. Maybe we want something like "Returns a `Credentials` object for logging in using...", possibly without the word "object" which is somewhat redundant. Or we could just leave the word "credentials" out entirely, since we don't need to repeat the signature in prose, and just say "Factory for logging in using ..."

Reply:
"credentials which authenticates" at the very least should be changed to "credentials which authenticate" (no `s`).

Reply:
I updated this to:
> Factory for `Credentials` which authenticate using the ...